test/pull/out/bar/transformation/keboola.python-transformation-v2/transformation-with-shared-code/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "parameters": {},
-  "shared_code_id": "%%TEST_SHARED_CODE_ID%%",
   "shared_code_row_ids": [
     "%%TEST_SHARED_CODE_ROW_ID%%"
-  ]
+  ],
+  "shared_code_path": "_shared/keboola.python-transformation-v2"
```

Comment:
Upraveny `pull` test, namiesto `id`, sa ulozi `path`.

Cele sa to robi preto, lebo ziadny subor v `local` nemoze obsahovat ID.
Ak by ho obsahoval, tak sa neda len tak skopirovat cely adresar do ineho projektu.
No ak tam je relativna path, tak to nie je problem, tak bude rovnaka.